### PR TITLE
Feature/logout

### PR DESCRIPTION
**Pull request by mahabaleshwarhnr**
- logout UI
- logout API
- clear cache

### CODE REVIEW

File: commons/utils/src/main/java/in/projecteka/jataayu/util/repository/CredentialsRepository.kt [hunk 1]
```diff
@@ -16,6 +17,7 @@ class CredentialsRepositoryImpl(private val sharedPreferences: SharedPreferences
     companion object {
         private const val AUTH_TOKEN = "AUTH_TOKEN"
         private const val CONSENT_TEMP_TOKEN = "CONSENT_TEMP_TOKEN"
+        private const val REFRESH_TOKEN = "CONSENT_TEMP_TOKEN"
```
Comment (AnusreeSajeevan) [inline commons/utils/src/main/java/in/projecteka/jataayu/util/repository/CredentialsRepository.kt]: REFRESH_TOKEN key value is same as  CONSENT_TEMP_TOKEN. 
Change it to something else

File: user/account/src/main/res/layout/fragment_user_account.xml [hunk 2]
```diff
@@ -71,13 +72,13 @@
...
-                            android:layout_alignParentEnd="true"
-                            android:layout_marginEnd="@dimen/content_margin_large"
-                            android:background="@drawable/ic_user"
-                            android:foreground="?android:attr/selectableItemBackground" />
+<!--                        <ImageView-->
+<!--                            android:layout_width="wrap_content"-->
+<!--                            android:layout_height="wrap_content"-->
+<!--                            android:layout_alignParentEnd="true"-->
+<!--                            android:layout_marginEnd="@dimen/content_margin_large"-->
+<!--                            android:background="@drawable/ic_user"-->
```
Comment (AnusreeSajeevan) [inline user/account/src/main/res/layout/fragment_user_account.xml]: Remove commented code

File: user/registration/src/main/java/in/projecteka/jataayu/registration/model/LoginRequest.kt [hunk 3]
```diff
@@ -3,7 +3,7 @@ package `in`.projecteka.jataayu.registration.model
 import com.google.gson.annotations.SerializedName
 
 data class LoginRequest(
-    @SerializedName("userName") val userName: String ,
+    @SerializedName("username") val userName: String ,
```
Comment (AnusreeSajeevan) [inline user/registration/src/main/java/in/projecteka/jataayu/registration/model/LoginRequest.kt]: This change already pushed to master. Please recheck this.

File: commons/utils/src/main/java/in/projecteka/jataayu/util/repository/CredentialsRepository.kt (hunk 1 above)
Comment (mahabaleshwarhnr) [reply commons/utils/src/main/java/in/projecteka/jataayu/util/repository/CredentialsRepository.kt:20]: okay. But later if they change logout feature will break.

**Review by AnusreeSajeevan — APPROVED** [review]
Looks good.

**Pull request merged**